public/docs/content/troubleshooting/parameters-between-stacks.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Unresolved resource dependencies error
+tags: cdk, stack
+packages: static-website, pipeline
+---
+
+# Passing parameters between CDK stacks will cause `Unresolved resource dependencies` error while trying to deploy
+
+## Links
+
+* [Cross stack values do not work with Source.data/jsonData [CDK github]](https://github.com/aws/aws-cdk/issues/19257)
+* [Nested stack runtime-config.json properties [PDK github]](https://github.com/aws/aws-prototyping-sdk/issues/84)
+
+## TLDR;
```

Comment:
You can also link to this for a solution: https://github.com/aws/aws-prototyping-sdk/issues/84

Reply:
second link points to that